WebView/ViewController.swift

```diff
@@ -20,25 +22,71 @@ class ViewController: UIViewController, WKNavigationDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
         navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Open", style: .plain, target: self, action: #selector(openTapped))
-        guard let url = URL(string: "https://www.redspace.com") else { return }
+        
+        let spacer = UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target:nil, action: nil)
+        let refresh = UIBarButtonItem(barButtonSystemItem: .refresh, target: webView, action: #selector(webView.reload))
+        let back = UIBarButtonItem(barButtonSystemItem: .rewind, target: webView, action: #selector(webView.goBack))
+        let forward = UIBarButtonItem(barButtonSystemItem: .fastForward, target: webView, action: #selector(webView.goForward))
+        
+        progressView = UIProgressView(progressViewStyle: .default)
+        progressView.sizeToFit()
+        let progressButton = UIBarButtonItem(customView: progressView)
+        
+        toolbarItems = [progressButton, spacer, back, forward, refresh]
+        navigationController?.isToolbarHidden = false
+        
+        webView.addObserver(self, forKeyPath: #keyPath(WKWebView.estimatedProgress), options: .new, context: nil)
+        
+        
+        guard let url = URL(string: "https://" + webSites[0]) else { return }
```

Comment:
Any reason this guard is moved here?
So generally it would makes more sense to put the `guard` statements at the top of the file so that you don't do any unnecessary work if the conditions for the guard are not met.

Reply:
I See. 
I was pushed down because I ended up adding more code on top. 
I'm fixing this right now.